Dockerfile

```diff
@@ -1,5 +1,6 @@
 FROM usgswma/python:3.8
 
+RUN apt-get update
```

Comment:
The base image has been updated, so I don't think this is strictly necessary anymore.